### DISCUSSION TITLE
How to record credit card expenses in instalments?

### DISCUSSION THREAD

**Discussion by beekingo** [firefly-iii/firefly-iii#4972] [Firefly III]
Hi there! First of all, many thanks to the developer of this excellent software.

I am trying to use it in Argentina to record all my expenses and I have a question with my credit cards.

Here it is common for credit card payments to be made in installments. For example, I buy something for $ 600 and pay it in 3 installments of $ 200.

I don't know what is the optimal way to register this in Firefly. Do I register a purchase of $ 200 each month? Do I record it as a $ 600 expense at the time of purchase?

I give you a minimized example of typical card movements.

01/01/2021 Purchase of $ 200 in 2 installments.
01/15/2021 Purchase of $ 500 in 1 payment.
01/30/2021 The summary of the card for $ 600 arrives
02/01/2021 I pay the total of the card for $ 600
02/30/2021 The $ 100 card summary arrives
03/01/2021 I pay the total of the card for $ 100

I appreciate any help.

**Comment by febriansasi** [comment]
Hi, I also have same problem, I use bill with each month for 3 month payment

**Comment by pogora** [comment]
Same here. Have you found a solution yet? 
The only way I found is rather complicated and thus of questionable use. 

Let's say we have payed -150 and 3x monthly rates.
1. Create a Tag  (i.e. installments)
2. Tag the Purchase in your Credit card account  -> Tag:installments -150
3. Split all the credit card payments to the monthly amount (50 ) -> Tag:installments | +50 | Balance -100
4. .. rinse and repeat

While it would be a way to track the installments it's just way to complicated. 
What I'd really like is a way to have this reflected in the Budget and thus influence the "Left to spend" Banner on the Dashboard.

Another alternative would be the use of the Liabilites Accounts, but I really dislike that I can't properly collect my expenses for that Store (expense account overview).

**Comment by MahmutHizal** [comment]
I am really saddened that such an important issue has gone unanswered and only received opinions from users. I condemn the developer. I hope the necessary adjustments/updates for installment transactions on credit cards will come as soon as possible.

**Reply by geomule** [reply]
I don't think condemning the developer is a good way to solve the problem :-)
Maybe asking the developer what are the difficulties in allowing such transactions to occur this way could be the right starting point? Maybe its a complicated feature to implement and maybe the developer doesn't have enough time.
